scripts/common/host-packages.sh

```diff
@@ -114,9 +114,19 @@ function _dpkg_install_host_packages() {
         return 0
     fi
 
-    DEBIAN_FRONTEND=noninteractive dpkg --install --force-depends-version --force-confold "${fullpath}"/*.deb
-
-    logSuccess "Host packages ${packages[*]} installed"
+    if DEBIAN_FRONTEND=noninteractive dpkg --install --force-depends-version --force-confold "${fullpath}"/*.deb; then
+        logSuccess "Host packages ${packages[*]} installed"
+    else
+        logFail "Unable to install the packages ${packages[*]}" >&2
+        if  commandExists docker && [[ "${packages[*]}" == *"containerd.io"* ]]; then
+            # If the error was faced because of conflicts it still possible to move forward because we will migrate docker to containerd and then, uninstall docker
+            # Usually, in this case containerd deps are installed already. Therefore, we just warning but we will not stop the install.
+            logWarn "Unable to install the containerd.io package probably because an compatible version of docker package is installed.\n
+            The script will continue and try to migrate the docker to containerd to perform this install. However, if it fails try to uninstall docker and re-run the script again.\n"
```

Comment:
Is this the best advice? If they uninstall docker in an airgapped environment how will they recover this images? I fear this will result in a support escalation.